conf/config.yaml

```diff
@@ -31,6 +31,12 @@
 #
 # If the configured environment variable can't be found, an error will be thrown.
 apisix:
+  node_listen:
+    - port: 9080                     # APISIX listening port
```

Comment:
We should provide the configuration separately in:
https://github.com/apache/apisix/blob/26a4ed4ad491f140fe441df281365c44723cae20/.travis/linux_openresty_common_runner.sh#L125

Reply:
done